### PR TITLE
Improve label visibility on really small areas

### PR DESCRIPTION
**Pull request by X9VoiD**
image

### CODE REVIEW

**Comment by AbstractQbit** [comment]
Maybe it would be better to just push the text a bit away from the rectangle, rather than hide it completely?

**Comment by InfinityGhost** [comment]
Resolve merge conflicts

**Comment by X9VoiD** [comment]
Merge conflicts resolved

File: OpenTabletDriver.UX/Controls/Area/AreaDisplay.cs [hunk 1]
```diff
@@ -217,22 +217,28 @@ private void DrawForeground(Graphics graphics, float scale)
 
         private void DrawRatioText(Graphics graphics, RectangleF area)
         {
+
```
Comment (InfinityGhost) [inline OpenTabletDriver.UX/Controls/Area/AreaDisplay.cs:220]: Remove this line

**Review by InfinityGhost — CHANGES_REQUESTED** [review]
One formatting issue but the rest looks good.

File: OpenTabletDriver.UX/Controls/Area/AreaDisplay.cs (hunk 1 above)
Comment (X9VoiD) [reply OpenTabletDriver.UX/Controls/Area/AreaDisplay.cs]: Fixed

**Review by InfinityGhost — APPROVED** [review]

**Pull request merged**